docs/walkthroughs/connecting-device-to-service.adoc

```diff
@@ -0,0 +1,48 @@
+[[mobile-core-public-ip]]
+= Mobile Core installation with public IP
+To start openshift cluster for local development and connect your device to same network you need to specify public IP of
+your laptop or server you want openshift being installed and provide following parameters
+```
```

Comment:
This  doesn't seem to show up when I view it for some reason

> To start openshift cluster for local development and connect your device to same network you need to specify public IP of your laptop or server you want openshift being installed and provide following parameters

Also I think it should say something like:

To use your local OpenShift cluster for mobile development using a real device rather than an emulator, you will need to specify  the public IP address of your local machine. 

You can do this during the setup of mobile using the install script in this repo: